scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py

```diff
@@ -109,10 +109,20 @@ def __init__(self, parent=None):
         self.electron_peaks = self._get_electron_peaks()
         self._generate_element_widgets()
 
-    # Return an unused colour, if all used then start with the first one
-    def get_color(self):
-        color = "C%d" % self.color_index
-        self.color_index = (self.color_index + 1) % self.num_colors
+    # Return the first unused colour, if all used then restart from the beginning of the cycle
```

Comment:
Could this be moved inside the function, and made into a [Python docstring](https://www.python.org/dev/peps/pep-0257/#id17)? Please also add a `:param element` in it that describes what that parameter is used for